lib/graphql/subscriptions/anycable_subscriptions.rb

```diff
@@ -103,7 +144,8 @@ def write_subscription(query, events)
           redis.mapped_hmset(SUBSCRIPTION_PREFIX + subscription_id, data)
           redis.sadd(SUBSCRIPTION_EVENTS_PREFIX + subscription_id, events.map(&:topic))
           events.each do |event|
-            redis.sadd(EVENT_PREFIX + event.topic, subscription_id)
+            redis.sadd(FINGERPRINTS_PREFIX + event.topic, event.fingerprint)
+            redis.sadd(SUBSCRIPTIONS_PREFIX + event.fingerprint, subscription_id)
```

Comment:
I think those sets should also be cleaned up in `delete_subscription`

Reply:
Agree. Added cleanup.